autosubmit/job/job.py

```diff
@@ -1263,6 +1263,12 @@ def retrieve_logfiles(self, platform: Any, raise_error: bool = False) -> Dict[st
                 raise AutosubmitCritical("Failed to retrieve logs for job {0}".format(self.name), 6000)
         else:
             self.write_stats(last_retrial)
+            if self.wrapper_type == "vertical":
+                for retrial in range(0, last_retrial + 1):
+                    Log.result(f"{platform.name}(log_recovery) Successfully recovered log for job '{self.name}' and retry '{retrial}'.")
```

Comment:
@dbeltrankyl why does it print the same message for each retry? Shouldn't it print only for the current retry? Or does it download the log files for each retry here? If so, maybe we could have one line that prints "recovered log for job ... and retries 0-N"? Just in case someone has >30 or so retries, to avoid getting too much on the logs output?

Reply:
Otherwise this change looks simple to review. Just need to understand if we really want to log N times the same message (where N is the number of retrials), or if we can summarize it. Looking at #2037 now.

Reply:
In the case of the vertical wrappers, it downloads all the retrials at once. 

In case you don't know or remember, I clarify that the vertical wrapper is doing the retrials inside the same wrapper without being resubmitted again.

Reply:
> If so, maybe we could have one line that prints "recovered log for job ... and retries 0-N"? Just in case someone has >30 or so retries,

I can apply that, was just to have the same format that in other cases

Reply:
>I can apply that, was just to have the same format as in the other cases ( easier to debug later ). In most of the cases, the job should only retry one or two times anyway

Sounds good to me. If we see that somewhere someone has too many log entries for this we can then create a new issue and reduce the output, but good argument on debugging/grepping later :+1: could be useful to search for a line of a specific retrial.

Thanks!